common/app/experiments/Experiments.scala

```diff
@@ -10,6 +10,15 @@ object ActiveExperiments extends ExperimentsDefinition {
   implicit val canCheckExperiment = new CanCheckExperiment(this)
 }
 
+object DCRFronts
```

Comment:
Did you mean to name this object `DCRFronts`?

Reply:
oops!